cylc/flow/scripts/lint.py

```diff
@@ -420,7 +565,7 @@ def get_reference_text(checks):
 def get_option_parser() -> COP:
     parser = COP(
         COP_DOC,
-        argdoc=[COP.optional(('DIR ...', 'Directories to lint'))],
+        argdoc=[COP.optional(('WORKFLOW | PATH ...', 'Workflows to lint'))],
```

Comment:
For the sake of consistency I would use https://github.com/cylc/cylc-flow/blob/8f5d16200cde477315725028a751dc8aacf0757b/cylc/flow/option_parsers.py#L48